op-challenger/scripts/alphabet/charlie.sh

```diff
@@ -1,22 +1,28 @@
 #!/bin/bash
 set -euo pipefail
 
-DIR=$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)
-cd "$DIR"
+SOURCE_DIR=$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)
+CHALLENGER_DIR=$(echo ${SOURCE_DIR%/*/*})
 
-DISPUTE_GAME_PROXY="0xB7f8BC63BbcaD18155201308C8f3540b07f84F5e"
+# Check that the fault game address file exists
+FAULT_GAME_ADDR_FILE="$CHALLENGER_DIR/.fault-game-address"
+if [[ ! -f "$FAULT_GAME_ADDR_FILE"]]; then
```

Comment:
Getting:
```
./scripts/alphabet/mallory.sh: line 9: syntax error in conditional expression: unexpected token `;'
./scripts/alphabet/charlie.sh: line 9: syntax error in conditional expression: unexpected token `;'
```
I think you need:
```suggestion
if [[ ! -f "$FAULT_GAME_ADDR_FILE" ]]; then
```
here and in mallory.